examples/output_voltage_measurement/pyproject.toml

```diff
@@ -0,0 +1,32 @@
+[tool.poetry]
+name = "output_voltage_measurement"
+version = "0.5.0"
+description = ""
+authors = ["National Instruments"]
+
+[tool.poetry.dependencies]
+python = "^3.8"
+ni-measurementlink-service = {version = "^1.1.0"}
+PyVISA = "^1.13.0"
+PyVISA-sim = "^0.5.1"
+nidcpower = { version = ">=1.4.4", extras = ["grpc"] }
+click = ">=7.1.2"
+grpcio = "*"
+
+[tool.poetry.group.dev.dependencies]
+mypy = ">=1.0"
+grpc-stubs = "^1.53"
+# Uncomment to use prerelease dependencies.
+# nidcpower = { git = "https://github.com/ni/nimi-python.git", subdirectory = "generated/nidcpower"}
```

Comment:
sorry, this is wrong in all the examples

```suggestion
# nidcpower = { git = "https://github.com/ni/nimi-python.git", subdirectory = "generated/nidcpower", extras = ["grpc"]}
```

Reply:
Sure, I updated this example's `pyproject.toml` now. Will update the other examples as a part of work item 2484653.